internal/pkg/template/workload.go

```diff
@@ -389,7 +389,7 @@ type PublishOpts struct {
 // Topic holds information needed to render a SNSTopic in a container definition.
 type Topic struct {
 	Name *string
-	Type string
+	Fifo Fifo
```

Comment:
Perhaps something like this:
```go
type Topic struct {
  // other fields...
  FIFOConfig *FIFOConfig
}
```
where non-nil would indicate that fifo is enabled